packages-ts/gauntlet-terra-contracts/src/commands/contracts/migration/index.ts

```diff
@@ -0,0 +1,84 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { TerraCommand, TransactionResponse } from '@chainlink/gauntlet-terra'
+import { AccAddress, MsgMigrateContract } from '@terra-money/terra.js'
+import { CONTRACT_LIST } from '../../../lib/contracts'
+
+type CommandInput<MigrationMessage> = {
+  contract: string
+  newCodeId: number
+  migrationMsg: MigrationMessage
+}
+
+export type MigrationMessageMaker<MigrationMessage> = (flags, args) => MigrationMessage
+
+export const makeMigrationCommand = <MigrationMessage>(
+  contract: CONTRACT_LIST,
+  makeMigrationMessage: MigrationMessageMaker<MigrationMessage>,
```

Comment:
I went a bit ahead, we'll need this command to be particular for each contract as every migration message will be different.
The only thing each contract command has to do is passing `makeMigratioMessage` that will construct the message based on the user input. 
I made an example on how it should look like with the OCR contract: https://github.com/smartcontractkit/chainlink-terra/pull/219/files#diff-f5e9f3694381cb803e8c4fa7798f04c6e87e3adaa92c9e9e6f184a19952a82c3R7